crates/sui-core/src/test_utils.rs

```diff
@@ -38,7 +38,7 @@ use sui_types::{
 use tokio::time::timeout;
 use tracing::{info, warn};
 
-const WAIT_FOR_TX_TIMEOUT: Duration = Duration::from_secs(15);
+const WAIT_FOR_TX_TIMEOUT: Duration = Duration::from_secs(5);
```

Comment:
Because of the new parameters this allows us to reduce the time we wait for transactions, as now Narwhal nodes will reconnect faster to each other in case of loss of connectivity

Reply:
can you please run `MSIM_TEST_NUM=20 cargo simtest` before merging this to make sure this isn't introducing flakiness.

Reply:
Very good point @mystenmark - I did run with `20` runs and indeed it did reveal some failures. I've increased the time to `10 seconds` instead and no problem. I'll leave like this and if someone still observes any failures we can always adapt.